### DISCUSSION TITLE
Postal Server unable to send test email on DigitalOcean

### DISCUSSION THREAD

**Discussion by mrkingsleyobi** [postalserver/postal#1704] [Help with using Postal]
I just finish install poster server on DigitalOcean(DO). Tried to send a test mail using smpter.net and I get SMTP send error.
![Screen Shot 2021-11-12 at 7 38 24 AM](https://user-images.githubusercontent.com/4938148/141421932-046c8884-32b0-4477-94c0-ddb4679e560a.png)
![Screen Shot 2021-11-12 at 7 38 10 AM](https://user-images.githubusercontent.com/4938148/141421949-ef97c4a9-770c-4502-94c6-cfe1e4f3deb7.png)
So to investigate the backend using the postal status I got this 
![Screen Shot 2021-11-12 at 7 22 32 AM](https://user-images.githubusercontent.com/4938148/141422145-3254ca47-5923-4174-9ab7-2972cb00815b.png)
I notice that postal_requeuer_1 and postal_worker_1 are at the state of Exit 1. I don't what that means if I need all to be UP.
Also I investigated if my firewall is active
![Screen Shot 2021-11-12 at 7 23 09 AM](https://user-images.githubusercontent.com/4938148/141422355-ed4b413a-b389-49d7-a964-d478c0e9b23c.png)
Which was not active. I don't know if I need that active. And finally I tried check the docker ps and I got this
![Screen Shot 2021-11-12 at 7 23 38 AM](https://user-images.githubusercontent.com/4938148/141422548-2efb2348-fe6f-410f-8c6f-1ad0b8b07174.png)
I do need help to fix the issues with sending mail.

Thanks

**Comment by willpower232** [comment]
You can use `postal logs requeuer` and `postal logs worker` to see why they failed. I bet it will be something related to rabbitmq, unfortunately it takes a while to get started so you might just need to try `postal stop` and `postal start`

Regarding why the message failed, you can have a look at `postal logs smtp` to see what message Postal sent to smtper
